src/Components/Cart-Card/Cart-Card.jsx

```diff
@@ -20,7 +21,14 @@ export const CartCard = () => {
     });
   return (
     <div class="all-cart-products">
-      {cart.length === 0 ? <h1>Please add Items in Cart</h1> : null}
+      {cart.length === 0 ? (
```

Comment:
If in else condition you are passing null, Better to use
```!cart.length && ()```

Reply:
Yes , thats a better solution , thanks for suggesting that